charts/nr-k8s-otel-collector/values.yaml

```diff
@@ -116,10 +116,16 @@ receivers:
     # -- (bool) Specifies whether the `prometheus` receiver is enabled
     # @default -- `true`
     enabled: true
+    # -- Sets the scrape interval for the `prometheus` receiver
+    # @default -- `20s`
+    scrapeInterval: 20s
```

Comment:
Why we are hardcoding this value here again? Is this default?

Reply:
this is so that the values file references (e.g.: `.Values.receivers.prometheus.scrapeInterval` ) has an actual value to reference